pkg/gui/confirmation_panel.go

```diff
@@ -84,7 +85,14 @@ func (gui *Gui) prepareConfirmationPanel(currentView *gocui.View, title, prompt
 }
 
 func (gui *Gui) onNewPopupPanel() {
-	gui.g.SetViewOnBottom("commitMessage")
+	_, err := gui.g.SetViewOnBottom("commitMessage")
+	if err != nil {
+		log.Fatal(err)
+	}
+	_, err = gui.g.SetViewOnBottom("pushPassUname")
```

Comment:
below you call _ = g.DeleteView("pushPassUname"), meaning that SetViewOnBottom could return an error, however you're logging with Fatal here which will crash the application. I believe you should either only delete the panel, or only move it to the bottom, not both